packages/plugin-audio-button-response/src/index.ts

```diff
@@ -274,6 +281,10 @@ class AudioButtonResponsePlugin implements JsPsychPlugin<Info> {
         btns[i].addEventListener("click", button_response);
       }
     }
+
+    return new Promise((resolve) => {
+      trial_complete = resolve;
+    });
```

Comment:
That's one way. Or just wrap the entire trial method with `return new Promise((resolve) => {` (to get rid of the extra variable and to get stricter typing)?